src/framework/components/rigid-body/system.js

```diff
@@ -580,44 +559,49 @@ class RigidBodyComponentSystem extends ComponentSystem {
      * query but within an array.
      * @param {Function} [options.filterCallback] - Custom function to use to filter entities.
      * Must return true to proceed with result. Takes the entity to evaluate as argument.
+     * @param {boolean} [options.findAll] - Whether to return all results. When false will return only
+     * the closest result. Defaults to false.
      *
-     * @returns {RaycastResult[]} An array of raycast hit results (0 length if there were no hits).
+     * @returns {HitResult[]} An array of raycast hit results (0 length if there were no hits).
      *
      * @example
      * // Return all results of a raycast between 0, 2, 2 and 0, -2, -2
-     * const hits = this.app.systems.rigidbody.raycastAll(new Vec3(0, 2, 2), new Vec3(0, -2, -2));
+     * const hits = this.app.systems.rigidbody.raycast(new Vec3(0, 2, 2), new Vec3(0, -2, -2), { findAll: true });
      * @example
      * // Return all results of a raycast between 0, 2, 2 and 0, -2, -2
      * // where hit entity is tagged with `bird` OR `mammal`
-     * const hits = this.app.systems.rigidbody.raycastAll(new Vec3(0, 2, 2), new Vec3(0, -2, -2), {
-     *     filterTags: [ "bird", "mammal" ]
+     * const hits = this.app.systems.rigidbody.raycast(new Vec3(0, 2, 2), new Vec3(0, -2, -2), {
+     *     filterTags: [ "bird", "mammal" ],
+     *     findAll: true
      * });
      * @example
      * // Return all results of a raycast between 0, 2, 2 and 0, -2, -2
      * // where hit entity has a `camera` component
-     * const hits = this.app.systems.rigidbody.raycastAll(new Vec3(0, 2, 2), new Vec3(0, -2, -2), {
-     *     filterCallback: (entity) => entity && entity.camera
+     * const hits = this.app.systems.rigidbody.raycast(new Vec3(0, 2, 2), new Vec3(0, -2, -2), {
+     *     filterCallback: (entity) => entity && entity.camera,
+     *     findAll: true
      * });
      * @example
-     * // Return all results of a raycast between 0, 2, 2 and 0, -2, -2
+     * // Return the closest result of a raycast between 0, 2, 2 and 0, -2, -2
```

Comment:
If those are vectors, we should wrap the elements in brackets, e.g. (0, 2, 2)

Reply:
Done